src/commands/dataconnect-sdk-generate.ts

```diff
@@ -42,10 +43,12 @@ export const command = new Command("dataconnect:sdk:generate")
         return;
       }
       for (const conn of serviceInfo.connectorInfo) {
+        const account = getProjectDefaultAccount(options.projectRoot);
```

Comment:
I think it's just `fdc build` and `fdc dev` that need the credentials (not `fdc generate`)

Reply:
Yeah, I figured we ought to provide them  as future proofing.